### DISCUSSION TITLE
Heart rate statistics and statistic graph with "hairline"

### DISCUSSION THREAD

**Discussion by pamatika** [OpenTracksApp/OpenTracks#1576] [Ideas]
Hi,

I tried my old Sports tracker ble heart rate monitor and it work fine. Good Work!
Would it possible to add simple statistics like average and max heart rates to stats tab? And then later maybe histogram/bar chart showing duration spent in in different heart rate ranges/zones. 50-70  71-100 etc...

The another wish. Would it be possible to add so called hairline feature to the graph? So when touching some point in the graph it would show the values in that point.

**Comment by dennisguse** [comment]
For the hairline feature: can you create an issue for this?

For the other things, there are already existing issues - just not time to do it.
